ip2asn.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+"""Takes ip addresses provided on stdin (one per line) and outputs info about the ASNs that contain them."""
+#Appears to work just fine under both python2 and python3.
+
+#Download https://iptoasn.com/data/ip2asn-combined.tsv.gz to current directory, gunzip it.
```

Comment:
Consider checking if the file does not exist and using `six.moves.urllib` (cross-compat) or `urllib.request` (python 3) to download the file.